1-1.js

```diff
@@ -0,0 +1,3 @@
+console.log("3" + true);
+console.log('5' + 5);
```

Comment:
Please follow unified code style: pick up to use single or double quotes. Please check another files.